moveit_setup_assistant/src/widgets/configuration_files_widget.cpp

```diff
@@ -489,6 +489,15 @@ bool ConfigurationFilesWidget::loadGenFiles()
   file.write_on_changes = MoveItConfigData::GROUPS;
   gen_files_.push_back(file);
 
+  // gazebo.launch ------------------------------------------------------------------
+  file.file_name_ = "gazebo.launch";
+  file.rel_path_ = config_data_->appendPaths(launch_path, file.file_name_);
+  template_path = config_data_->appendPaths(template_launch_path, "gazebo.launch");
+  file.description_ = "gazebo launch file";
```

Comment:
Please capitalize this description and explain in a sentence what it does, see the other examples in this file.

Also apply the same change to the ros_controllers.launch you added